Source/Scene/ImplicitSubtree.js

```diff
@@ -509,24 +511,41 @@ function preprocessBufferViews(bufferViewHeaders, bufferHeaders) {
 function markActiveBufferViews(subtreeJson, bufferViewHeaders) {
   let header;
   const tileAvailabilityHeader = subtreeJson.tileAvailability;
-  if (defined(tileAvailabilityHeader.bufferView)) {
+  if (defined(tileAvailabilityHeader.bitstream)) {
+    header = bufferViewHeaders[tileAvailabilityHeader.bitstream];
+  } else if (defined(tileAvailabilityHeader.bufferView)) {
```

Comment:
Same here, in this function comment explaining that `bufferView` was from an older schema draft.